content/the-ember-times-issue-198.md

```diff
@@ -14,52 +15,46 @@ tags:
 
 ---
 
-## [🐹 1. Section title in sentence case](section-url)
-
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
-
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+## [🗳️ 1. Fill out the Ember Community Survey by April 7th!](https://emberjs.com/survey/2022/)
+  
+The annual [Ember Community Survey](https://emberjs.com/survey/2022/) helps guide direction in the Ember ecosystem.
+Data informs project prioritization, it sparks new ideas, and reveals opportunities for growth.
+The Ember core teams would love to hear from you!
+  
+The survey is primarily intended for developers who use the Ember JavaScript framework, either for work or personal use, although any developer is welcome to fill out this survey. The survey should take approximately 15 minutes to complete in full, though all questions are optional.
+  
+[Learn more about the survey and fill it out](https://emberjs.com/survey/2022/) by April 7th!
 
 ---
 
-## [🐹 2. Section title in sentence case](section-url)
+## [📽️ 2. Remote Ember.js Europe Meetup, featuring Ember Data](https://twitter.com/embereurope)
```

Comment:
```suggestion
## [📽️ 2. Hybrid Ember.js Europe Meetup, featuring Ember Data](https://twitter.com/embereurope)
```

The idea behind Ember.js Europe is it's a hybrid event once a quarter from a different location in Europe each time (this time Dublin) as well as remote via Zoom.